### PR TITLE
add: docs manual/geth-lodestar

### PR DESCRIPTION
**Pull request by atkinsonholly**
Added manual setup file for Geth-Lodestar

### CODE REVIEW

File: manual/setup-geth-lodestar.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+mkdir testnet-all
+cd testnet-all
+```
+
+Download the `testnet-all.tar.gz` file for the [latest release of the Ephemery testnet](https://github.com/ephemery-testnet/ephemery-genesis/releases) to this directory using `wget`. Then unzip inside this folder.
+```
+tar -xzf testnet-all.tar.gz
+```
+
+### Generate jwt
```
Comment (taxmeifyoucan) [inline manual/setup-geth-lodestar.md]: just to note that clients currently create jwt token themselves, might be left as optional

File: manual/setup-geth-lodestar.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+```
+openssl rand -hex 32 | tr -d "\n" > "/tmp/jwtsecret"
+```
+
+### Execution Layer
+
```
Comment (taxmeifyoucan) [inline manual/setup-geth-lodestar.md:35]: I would give user the option whether to compiler or just download a binary. And for compilation, just refer to geth docs

File: manual/setup-geth-lodestar.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+     --networkid {networkID} \
+     --syncmode=full \
+     --port 30303 \
+     --http \
+     --datadir "datadir-geth" \
+     --authrpc.jwtsecret=/tmp/jwtsecret \
+     --bootnodes {bootnodes}
+```
+
+For `{bootnodes}` look in ~/testnet-all/boot_enode.txt. Entries must be separated,by,commas and "enclosed in quotes".
```
Comment (taxmeifyoucan) [inline manual/setup-geth-lodestar.md:66]: User can just source the file with variables, load it to the shell session and run the same command 

```
source node_vars.txt 
...
--bootneds $BOOTNODE_ENR_LIST
```

File: manual/setup-geth-lodestar.md (hunk 1 above)
Comment (atkinsonholly) [reply manual/setup-geth-lodestar.md]: Ok! I reflected the instructions given in the geku file. Happy to edit both to make this clear

File: manual/setup-geth-lodestar.md (hunk 2 above)
Comment (atkinsonholly) [reply manual/setup-geth-lodestar.md:35]: Same comment - I'm happy to add this to both files for Geth

File: manual/setup-geth-lodestar.md (hunk 3 above)
Comment (atkinsonholly) [reply manual/setup-geth-lodestar.md:66]: Same - will update both

**Pull request merged**